Makefile

```diff
@@ -99,27 +99,40 @@ release: \
 
 release-victoria-metrics: \
 	release-victoria-metrics-amd64 \
+	release-victoria-metrics-linux-amd64 \
+	release-victoria-metrics-darwin-amd64 \
 	release-victoria-metrics-arm \
+	release-victoria-metrics-linux-arm \
 	release-victoria-metrics-arm64 \
```

Comment:
This target needs to be removed, since it is overridden by `release-victoria-metrics-linux-arm64`